frontend/packages/ceph-storage-plugin/src/components/ocs-install/internal-mode/install-wizard-steps/select-capacity-and-nodes-step.tsx

```diff
@@ -42,6 +59,110 @@ const validate = (scName, enableMinimal, enableFlexibleScaling): ValidationType[
   return validations;
 };
 
+const isArbiterDisabled = (nodesData: NodeKind[], validateSelectedNodes?: boolean) => {
+  const validation = {
+    nodesLen: validateSelectedNodes ? 4 : 5,
+    zoneLen: validateSelectedNodes ? 2 : 3,
+  };
+  const uniqZones: Set<string> = new Set(nodesData.map((node) => getZone(node)));
+  if (nodesData.length < validation.nodesLen) return true;
+  if (uniqZones.size !== validation.zoneLen) return true;
+  const zonePerNode = countNodesPerZone(nodesData);
+  let count = 0;
+  Object.keys(zonePerNode).forEach((zone) => {
+    if (zonePerNode[zone] >= 2) count += 1;
+  });
```

Comment:
```suggestion
const count = Object.keys(zonePerNode).reduce((acc, curr) => {
    if(zonesPerNode[zone] > =2) acc += 1;
    return acc;
    }, 0);
```
Let's use another name for `count`. Something that would give more insight on actually what we are counting.

Reply:
eslint throws and error when you do the assignment to function parameter.

`Assignment to function parameter 'acc' no-param-reassign`